schema.prisma

```diff
@@ -40,6 +40,7 @@ model Person {
   updatedAt          DateTime?           @updatedAt @map("updated_at") @db.Timestamptz(6)
   // Receive marketing notifications
   newsletter         Boolean?            @default(false)
+  helpUsImporve      Boolean?            @default(false)
```

Comment:
There is a typo. Change to `helpUsImprove`
We use snake_case for referencing a field via raw SQL query. Would you please add @map("help_us_improve") at the end of this line. 
Example:
```js
helpUsImporve      Boolean?            @default(false) @map("help_us_improve")
```